### PR TITLE
Text input field width is now to set to `full` to fit the screen width

### PR DESCRIPTION
**Pull request by LornaHarwood**
## Changes in this PR

Fixes text input fields width and now displays to full width

### Before

Screen Shot 2021-04-27 at 17 14 07

### After

Screen Shot 2021-04-27 at 17 13 46

### CODE REVIEW

File: app/views/steps/long_text.html.erb [hunk 1]
```diff
@@ -2,7 +2,7 @@
   <%= f.govuk_text_area :response,
     label: { text: @step_presenter.title, size: "l" },
     hint: -> { @step_presenter.help_text_html },
-    width: "one-third",
+    width: "full",
     rows: 9
```
Comment (tahb) [inline app/views/steps/long_text.html.erb:6]: The width looks good but the height seems to have an existing issue. I wonder why this isn't 9 rows high?

Comment (tahb) [reply app/views/steps/long_text.html.erb:6]: Oh I think I see what's confused me. This change is for long-text but screenshots from short-text have been used?

**Review by tahb — APPROVED** [review]
LGTM after initial screenshot confusion. Needs a line in our CHANGELOG, then this can be merged.

File: app/views/steps/long_text.html.erb (hunk 1 above)
Comment (LornaHarwood) [reply app/views/steps/long_text.html.erb:6]: Yes, sorry, I should have been more explicit at what the screenshots are showing: this is just the short-text display

**Pull request merged**